src/view/components/extensionViewForm.jsx

```diff
@@ -1,76 +1,96 @@
 import React, { useEffect, useContext, useRef } from "react";
 import { useFormik, FormikProvider } from "formik";
 import PropTypes from "prop-types";
-import wrapValidateWithErrorLogging from "../utils/wrapValidateWithErrorLogging";
+import { object } from "yup";
 import ExtensionViewContext from "./extensionViewContext";
+import useReportAsyncError from "../utils/useReportAsyncError";
 
-const ExtensionViewForm = ({
-  initialValues,
-  getSettings,
-  validateFormikState = () => undefined,
-  formikStateValidationSchema,
-  validateNonFormikState = () => true,
-  render
-}) => {
-  const formikProps = useFormik({
-    initialValues,
-    enableReinitialize: true,
```

Comment:
I had misunderstood what `enableReinitialize` was for when I had set it to `true`. I thought it meant that it would allow us to re-initialize the form using `resetForm`, but it actually means that if `initialValues` changes, that the form will automatically reset. In our case, whenever we reset the form, we explicitly use `resetForm`. What I discovered is that there's a bug in Formik when we call `resetForm` on our first render (as now happens on SendEvent, for example). Formik resets its values to whatever we pass into `resetForm`, but then it resets its values again back to what `initialValues` was, which is `undefined` in our case (related to https://github.com/formium/formik/issues/2440). Fortunately, we don't need/want `enableReinitialize: true`, so removing it also avoids the bug.